tests/integrations/config/mod.rs

```diff
@@ -279,15 +279,15 @@ fn test_serde_custom_tikv_config() {
             soft_pending_compaction_bytes_limit: ReadableSize::gb(12),
             hard_pending_compaction_bytes_limit: ReadableSize::gb(12),
             titan: TitanCfConfig {
-                min_blob_size: ReadableSize::gb(4).0 as u64, // disable titan default
+                min_blob_size: 1024, // default value
```

Comment:
Please use non-default value in test

Reply:
This is for non-default CFs. The min_blob_size here reflect default value since we don't set their value from test-custom.toml. And it doesn't make sense to set them.